From db6a0079abdce13a96e5a5c6e37390869c5a7b56 Mon Sep 17 00:00:00 2001
Message-Id: <db6a0079abdce13a96e5a5c6e37390869c5a7b56.1350312451.git.minovotn@redhat.com>
In-Reply-To: <aa2da19f266f7dd7345db5620ee362446fc6e806.1350312451.git.minovotn@redhat.com>
References: <aa2da19f266f7dd7345db5620ee362446fc6e806.1350312451.git.minovotn@redhat.com>
From: Alon Levy <alevy@redhat.com>
Date: Sun, 7 Oct 2012 15:31:53 +0200
Subject: [PATCH 16/27] qxl: ioport_write: remove guest trigerrable abort

RH-Author: Alon Levy <alevy@redhat.com>
Message-id: <1349623920-19894-10-git-send-email-alevy@redhat.com>
Patchwork-id: 42784
O-Subject: [PATCH RHEL-6.4 v2 09/16] qxl: ioport_write: remove guest trigerrable abort
Bugzilla: 770842
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>
RH-Acked-by: Arnon Gilboa <agilboa@redhat.com>

Signed-off-by: Alon Levy <alevy@redhat.com>
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>

Upstream: 4763e2cadd3b2ecfc41e83b99b88b8a68d2e6055
---
 hw/qxl.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 hw/qxl.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/hw/qxl.c b/hw/qxl.c
index 4515314..80b61c0 100644
--- a/hw/qxl.c
+++ b/hw/qxl.c
@@ -1500,8 +1500,7 @@ async_common:
         qxl_spice_destroy_surfaces(d, async);
         break;
     default:
-        fprintf(stderr, "%s: ioport=0x%x, abort()\n", __FUNCTION__, io_port);
-        abort();
+        qxl_guest_bug(d, "%s: unexpected ioport=0x%x\n", __func__, io_port);
     }
     return;
 cancel_async:
-- 
1.7.11.7

